samples/java-spring-cinema-booking-choreography/src/test/java/com/example/cinema/ShowBuilder.java

```diff
@@ -0,0 +1,48 @@
+package com.example.cinema;
+
+import com.example.cinema.model.Show;
+import io.vavr.Tuple2;
+import io.vavr.collection.HashMap;
+import io.vavr.collection.Map;
+
+import java.util.List;
+
+import static com.example.cinema.DomainGenerators.randomPrice;
+import static com.example.cinema.DomainGenerators.randomShowId;
+import static com.example.cinema.model.Show.ShowCreator.createSeats;
+
+public class ShowBuilder {
+
+  final static int MAX_SEATS = 100;
+  private String id = randomShowId();
+  private String title = "Random title";
```

Comment:
this title doesn't seem to be that random

Reply:
Maybe seed is wrong